perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -149,8 +158,21 @@ def _IsReady(self):
     Returns:
       True if the resource was ready in time, False if the wait timed out.
     """
+    cmd = util.GcloudCommand(self, 'sql', 'instances', 'describe',
```

Comment:
Question: when is _IsReady() being called? During prepare phase or provision phase?

The reason it's important is becuase we also have a data loading step at the prepare phase, if "is ready" implies the data is loaded and prepare is done, then we are not ready here yet. I think it's for the provision phase only, but I could be wrong here.

@gareth-ferneyhough FYI.